### ISSUE TITLE
.gitignore ignores *.out files from regression tests

### ISSUE THREAD

**Issue by andreasscherbaum** [greenplum-db/gpdb#594]
.gitignore has a rule for globally ignoring *.out files.
This also ignores all output files for regression tests.

**Comment by danielgustafsson** [comment]
This was fixed in #585, did that not solve the problem?

**Comment by andreasscherbaum** [comment]
You beat me with your fix. I found it yesterday, and logged it today, and you fixed it already ;-)

Thanks for that, can confirm that git sees the files now:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        src/test/regress/expected/xml_2.out
```
